### ISSUE TITLE
Using another loss functions?

### ISSUE THREAD

**Issue by Helaly96** [nimarb/pytorch_influence_functions#36]
In the case i want to use another loss function than NLL, for example a regression problem.
i will simply change the calc_loss function implementation.
correct?
